_includes/scripts.html

```diff
@@ -57,3 +59,51 @@
 	<script>document.write('<script src="http://' + (location.host || 'localhost').split(':')[0] + ':35729/livereload.js?snipver=1"></' + 'script>')</script>
 {% endif %}
 
+<script>
```

Comment:
this will need to live in a separate .js file to avoid loading it on every page. Have a look at https://github.com/p2pu/p2pu-website/blob/gh-pages/en/publications/roundtables.html#L4-L6